diff --git a/node_modules/@capacitor-community/contacts/Package.swift b/node_modules/@capacitor-community/contacts/Package.swift
index 6be3b7d..f85286a 100644
--- a/node_modules/@capacitor-community/contacts/Package.swift
+++ b/node_modules/@capacitor-community/contacts/Package.swift
@@ -10,7 +10,7 @@ let package = Package(
             targets: ["ContactsPlugin"])
     ],
     dependencies: [
-        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", from: "7.0.0")
+        .package(url: "https://github.com/ionic-team/capacitor-swift-pm.git", from: "8.0.0")
     ],
     targets: [
         .target(
